src/source_widget.py

```diff
@@ -0,0 +1,46 @@
+from rich.syntax import Syntax
+
+from textual.app import ComposeResult
+from textual import events
+from textual.containers import Container, VerticalScroll
+from textual.geometry import Region
+from textual.widgets import Static
+from styles import HIGHLIGHT
+
+from events import HoverLine
+
+
+class SourceWidget(Container):
+
+    def compose(self) -> ComposeResult:
+        with VerticalScroll(classes="scroller"):
+            # To use an editor
+            # yield TextArea.code_editor("", language="python", classes="editor")
```

Comment:
Did yo intend to delete this comment?

Reply:
Yes, thanks for the catch. I've removed both.